python/cudf/cudf/tests/test_index.py

```diff
@@ -2897,3 +2897,16 @@ def test_scalar_getitem(self, index_values, i):
         assert not isinstance(index[i], cudf.Index)
         assert index[i] == index_values[i]
         assert_eq(index, index.to_pandas())
+
+
+def test_index_mixed_dtype_error():
+    pi = pd.Index(
+        [
+            pd.Timestamp("1970-01-01 00:00:00.000000001"),
+            pd.Timestamp("1970-01-01 00:00:00.000000002"),
```

Comment:
Could you test with `pd.Timedelta` too? (We had a similar issue with Timedelta not so long ago)

Reply:
Done 👍